e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -444,6 +444,63 @@
               "createOptions": ""
             }
           },
+          "deploymentTest1": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "DEPLOYMENT_TESTER_MODE": {
```

Comment:
Prefix with IOTEDGE_

Reply:
Why do we want to add IOTEDGE_ Prefix; should it be used only when it is added by edgeAgent?  This "DEPLOYMENT_TESTER_MODE" is a custom env variable for DeploymentTester module only.

Reply:
I think you are right, the idea to add IOTEDGE_ was not to conflict with a var set by the custom module.